opensrp-chw/src/kk/assets/recurring_service_types.json

```diff
@@ -1498,5 +1498,176 @@
         }
       }
     ]
+  },
+  {
+    "type": "Child safety",
+    "service_logic": "",
+    "service_group": "child",
+    "openmrs_service_name": {
+      "parent_entity": "",
+      "entity": "concept",
+      "entity_id": ""
+    },
+    "openmrs_date": {
+      "parent_entity": "",
+      "entity": "encounter",
+      "entity_id": "encounter_date"
+    },
+    "services": [
+      {
+        "id": "160",
+        "name": "Child safety1",
+        "schedule": {
+          "due": {
+            "reference": "dob",
+            "offset": "+3m"
```

Comment:
Is this visit 6 at 3 months? If so we might need to standardize it. Based on the conversation I put 2 months here.

Reply:
You are right, this was before the conversation we had